man.Rmd

```diff
@@ -8,34 +8,61 @@ library(stringr) # for links
 Base R provides a standard way of documenting a package where each function is documented in a **topic,** an `.Rd` file ("R documentation") in the `man/` directory.
 `.Rd` files use a custom syntax, loosely based on LaTeX, and can be rendered to HTML, plain text, or pdf, as needed, for viewing in different contexts.
 
 In the devtools ecosystem, we don't edit `.Rd` files directly with our bare hands.
-Instead, we include specially formatted comments *above* the source code for each function.
-Then we use the roxygen2 package to generate the `.Rd` files from these special "roxygen comments".
-There are a few advantages to using roxygen2:
+Instead, we include specially formatted "roxygen comments" above the source code for each function[^man-1].
+Then we use the roxygen2 package to generate the `.Rd` files from these special comments[^man-2]
+. There are a few advantages to using roxygen2
+:
+
+[^man-1]: The name "roxygen" is a nod to the Doxygen documentation generator, which inspired the development of an R package named roxygen.
+    Then that original concept was rebooted as roxygen2, similar to ggplot2.
+
+[^man-2]: The NAMESPACE file is also generated from these roxygen comments (or, rather, it *can* be and that is the preferred devtools workflow).
+    The NAMESPACE file is covered in sections @sec-metadata-namespace and @sec-dependencies-namespace.
 
 -   Code and documentation are co-located, so that when you modify your code, it's easy to remember to also update your documentation.
 
--   You can use markdown, rather than having to learn a new text formatting syntax.
+-   You can use markdown, rather than having to learn a one-off markup language that only applies to `.Rd` files.
+    In addition to formatting, the automatic linking functionality gives you lots of cross-references "for free".
 
 -   There's a lot of `.Rd` boilerplate that's automated away.
 
 -   roxygen2 provides a number of tools for sharing content across documentation topics and even between topics and vignettes.
 
-In this chapter we'll focus on documenting functions, but the same ideas apply to documenting datasets, classes and generics, and packages.
-You can learn more about those important topics in `vignette("rd-other", package = "roxygen2")`.
+In this chapter we'll focus on documenting functions, but the same ideas apply to documenting datasets (@sec-documenting-data), classes and generics, and packages.
+You can learn more about those important topics in `vignette("rd-other", package = "roxygen2")`[^man-3].
+
+[^man-3]: The text `vignette("rd-other", package = "roxygen2")` is actually a great example of autolinking in the devtools ecosystem.
+    This is literally the R code you would execute in the console to navigate to a specific vignette within the roxygen2 package.
+    But in many rendered contexts, it automatically becomes a hyperlink to that same vignette in roxygen2's pkgdown website.
```

Comment:
I really want to explain this `vignette("name", package = "package")` syntax. But perhaps I can move this footnote to the to-be-writte section I want to add for "[R] Markdown + roxygen2 Greatest Hits".

Reply:
Moved